roles-in-tech/qa.md

```diff
@@ -2,89 +2,105 @@
 
 ### Projected Time
 
-Example: 30-45 minutes
-- Lesson: ? min
-- Guided Practice: ?-? min
-- Independent Practice: ? min
-- Check for Understanding: ?-? min
+- Lesson: 20 min
+- Guided Practice: 10 min
+- Independent Practice: 12 min
+- Check for Understanding: 15 min
 
 ### Prerequisites
 
-Here are links to lessons that should be completed before this lesson:
-
-- Example: [Git version control](version-control/git-version-control/git-version-control.md)
+None
```

Comment:
Search for any outlines with “testing” in the title and see if they would make good prereqs.